tests/functional/cylc-install/03-file-transfer.t

```diff
@@ -52,7 +52,7 @@ mkdir .git .svn dir1 dir2
 touch .git/file1 .svn/file1 dir1/file1 dir2/file1 file1 file2
 run_ok "${TEST_NAME}" cylc install "${RND_SUITE_NAME}" --no-run-name
 
-tree -a -v -I '*.log|03-file-transfer*' "${RND_SUITE_RUNDIR}/" > 'basic-tree.out'
+tree -a -v -I '*.log|03-file-transfer*' --charset=UTF8 "${RND_SUITE_RUNDIR}/" > 'basic-tree.out'
```

Comment:
Also to note, `--noreport` means you can leave out the `7 directories, 5 files`

Reply:
This may become useful for my branch - the output is dependent on which plugins are installed. :(

Reply:
You can use globs to limit the output, e.g. `tree ~/cylc-run/foo/something-*`